### PR TITLE
feat: add --version to cli

### LINKED ISSUE #171: Unknown option
Instructions to check version doesn't return expected result.

After executing `open-rpc-generator-client --version` (inside README.md line 33)

we get this result in Ubuntu 18.04

`error: unknown option --version`

### PR DESCRIPTION
**Pull request by devonwesley**
fixes #171

### CODE REVIEW

**Comment by BelfordZ** [comment]
this breaks the build

**Comment by BelfordZ** [comment]
hmm I guess I am wrong! I seem to recall having already tried this and when published it was broke, but I dont remember now. Lets just roll with it!

**Comment by BelfordZ** [comment]
Ah right, it works because you are using `require` ! got it!

File: src/cli.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import orpcGenerator from "./";
 import { parseOpenRPCDocument } from "@open-rpc/schema-utils-js";
 
 program
+  .version(`v${require('../../package.json').version}`, '-v, --version')
```
Comment (BelfordZ) [inline src/cli.ts]:
```suggestion
  .version(`${require('../../package.json').version}`, '-v, --version')
```

as documented in pristine, we never prefix versions with `v`

**Comment by BelfordZ** [comment]
@devonwesley when this is done, could you also update the other OpenRPC projects that include CLI to also use this pattern?

in some of them, we still havent converted the cli to TS.

**Comment by devonwesley** [comment]
@BelfordZ Yeah, No worries. I'll go through the projects and update them the same way.

**Pull request merged**